server/routes/routerRequestHandlers/retrieveActionPlanReviews.test.ts

```diff
@@ -10,6 +10,13 @@ import ActionPlanReviewStatusValue from '../../enums/actionPlanReviewStatusValue
 
 jest.mock('../../services/reviewService')
 
+const reviewJourneyEnabledForPrison = jest.fn()
```

Comment:
Not strictly related to this PR, but improved the tests here, specifically how the tests test the 2 different code paths for when the `reviewJourneyEnabledForPrison` feature toggle is enabled or not